docs/source/Install/installOnWindows.rst

```diff
@@ -17,10 +17,10 @@ In order to run Basilisk, the following software will be necessary:
    program from the command line
 -  `Python <https://www.python.org/downloads/windows/>`__ 3.8.x or greater
 -  `pip <https://pip.pypa.io/en/stable/installing/>`__
--  Visual Studios 15 or Greater
+-  Visual Studios 15 or greater
```

Comment:
I wonder if this is a good opportunity to change the default Windows compiler to Visual Studio Community 2022?

Reply:
I agree we should be upgrading the default Visual Studio version, but not in this branch.  I see that as a separate branch as we will have to do many tests on this across computers.

Reply:
Yes this change will require us to check CI runs as well. Let's deal with it in a separate block of work. I've created an issue to track this #532.